shaky/src/main/java/com/linkedin/android/shaky/FeedbackActivity.java

```diff
@@ -63,6 +67,8 @@ public void onCreate(Bundle savedInstanceState) {
 
         imageUri = getIntent().getParcelableExtra(SCREENSHOT_URI);
         userData = getIntent().getBundleExtra(USER_DATA);
+        // Defaults to paper plane icon
```

Comment:
I would not add this comment as the code explains itself.
I would also move this into the fragment. Activity does not need to know about the default resource for the icon

Reply:
How would you add it to the fragment? getIntExtra forces a default value and I dont know another way to get the extra from the bundle without using that method -- I could pass in the whole Intent but the pattern seems to just get the extras out, save them in a variable, and pass them into the fragment?

Reply:
In this case, make it a package variable within the fragment and reference to it.